packages/remix-react/routeModules.ts

```diff
@@ -115,7 +114,7 @@ export interface V1_HtmlMetaDescriptor {
 export type HtmlMetaDescriptor = V1_HtmlMetaDescriptor;
 
 export type V2_HtmlMetaDescriptor =
-  | { charSet: "utf-8" }
+  | { charSet: "utf-8"; charset?: "utf-8" }
```

Comment:
This tells typescript that we accept an object with `charSet=utf-8`, and we also accept in that same object, an optional lowercased version.
This doesn't really make sense.

Reply:
```suggestion
  | { charSet: "utf-8" }
```